Makefile

```diff
@@ -140,17 +141,25 @@ ui:
 .PHONY: install
 install: check-dcos
 	$(DCOS_CMD) package install --yes marathon-lb;\
-	$(DCOS_CMD) package install --yes kubernetes --package-version=$(KUBERNETES_FRAMEWORK_VERSION) --options="$(PATH_TO_PACKAGE_OPTIONS)";\
-	$(DCOS_CMD) marathon app add "$(PWD)/.deploy/kubeapi-proxy.json"
+	$(DCOS_CMD) marathon app add "$(PWD)/.deploy/kubeapi-proxy.json" || true
+	$(DCOS_CMD) package install --yes kubernetes --package-version="$(KUBERNETES_FRAMEWORK_VERSION)";\
+	sleep 30
+	$(DCOS_CMD) kubernetes cluster create --yes --options="$(PATH_TO_PACKAGE_OPTIONS)";\
+
 
 .PHONY: watch
 watch:
-	watch dcos kubernetes plan show deploy
+	watch dcos kubernetes cluster debug --cluster-name=dev/kubernetes01 plan show deploy
```

Comment:
Can we make `dev/kubernetes01` a variable ? Why do we deploy it within a group `dev/kubernetes01` ?

Reply:
We can and should, but it is propagated to other files as well (e.g. `resources/options.json.gcp`). I'll take a note out of that and address in a follow-up PR if that is OK.

Reply:
I can do that follow up. The reason why I chose this naming was to be consistent with other documentation we have which all relies on dev/kubernetes01 as the first cluster. I think it makes sense to keep this it also teaches the user a very import fact that this is how things are put into folders via the service page and an idea about the internal dns names.